modules/Bio/EnsEMBL/Production/Pipeline/GPAD/LoadFile.pm

```diff
@@ -34,96 +34,25 @@ use Bio::EnsEMBL::Analysis;
-            JOIN transcript tf USING (transcript_id)
-            JOIN seq_region s USING (seq_region_id)
-            JOIN coord_system c USING (coord_system_id)
-            WHERE x.external_db_id=1000
-            AND c.species_id=?
-            AND (x.info_type="PROJECTION"
-            OR  x.info_type="DIRECT" OR x.info_type = "DEPENDENT")',
-
-        # Delete by analysis.logic_name='go_projection' & 'interpro2go'
-        # interpro2go annotations should be superceded by GOA annotation
-        'DELETE ox.*,onx.*,dx.*  FROM xref x
-                      JOIN object_xref ox USING (xref_id)
-                      LEFT JOIN ontology_xref onx USING (object_xref_id)
-                      LEFT JOIN dependent_xref dx USING (object_xref_id)
-                      JOIN analysis a USING (analysis_id)
-                      JOIN translation tl ON (ox.ensembl_id=tl.translation_id)
-                      JOIN transcript tf USING (transcript_id)
-                      JOIN seq_region s USING (seq_region_id)
-                      JOIN coord_system c USING (coord_system_id)
-                      WHERE x.external_db_id=1000
-                      AND c.species_id=?
-          AND (a.logic_name="goa_import"
-          OR a.logic_name="interpro2go")',
-
-        # Same deletes but for GOs mapped to transcripts
-       'DELETE ox.*,onx.*,dx.*  FROM xref x
-                      JOIN object_xref ox USING (xref_id)
-                      LEFT JOIN ontology_xref onx USING (object_xref_id)
-                      LEFT JOIN dependent_xref dx USING (object_xref_id)
-                      JOIN analysis a USING (analysis_id)
-                      JOIN transcript tf ON (ox.ensembl_id = tf.transcript_id)
-                      JOIN seq_region s USING (seq_region_id)
-                      JOIN coord_system c USING (coord_system_id)
-                      WHERE x.external_db_id=1000
-                      AND c.species_id=?
-                      AND (x.info_type="PROJECTION"
-                      OR  x.info_type="DIRECT" OR x.info_type = "DEPENDENT")',
-
-        'DELETE ox.*,onx.*,dx.*  FROM xref x
-                      JOIN object_xref ox USING (xref_id)
-                      LEFT JOIN ontology_xref onx USING (object_xref_id)
-                      LEFT JOIN dependent_xref dx USING (object_xref_id)
-                      JOIN analysis a USING (analysis_id)
-                      JOIN transcript tf ON (ox.ensembl_id = tf.transcript_id)
-                      JOIN seq_region s USING (seq_region_id)
-                      JOIN coord_system c USING (coord_system_id)
-                      WHERE x.external_db_id=1000
-                      AND c.species_id=?
-                      AND (a.logic_name="goa_import"
-                      OR a.logic_name="interpro2go")'];
-
-      for my $sql (@$sqls) {
-	$self->log()->debug("Executing $sql");
-	$dba->dbc()->sql_helper()->execute_update(-SQL=>$sql);
-      }
+      $self->cleanup_GO($dba);
```

Comment:
As mentioned above, I don't think this is needed anymore.

Reply:
As mentioned above. I will move it back to loadFile module

Reply:
see latest commit